src/analytics/ledger/active_addresses.rs

```diff
@@ -5,29 +5,34 @@ use std::collections::HashSet;
 
 use time::{Duration, OffsetDateTime};
 
-use super::{AddressCount, TransactionAnalytics};
-use crate::types::{
-    ledger::{LedgerOutput, LedgerSpent, MilestoneIndexTimestamp},
-    stardust::block::Address,
+use super::TransactionAnalytics;
+use crate::{
+    db::collections::analytics::DailyActiveAddressAnalyticsResult,
+    types::{
+        ledger::{LedgerOutput, LedgerSpent, MilestoneIndexTimestamp},
+        stardust::block::Address,
+    },
 };
 
 /// Computes the number of addresses that were active during a given time interval.
-pub struct ActiveAddresses {
-    start_time: OffsetDateTime,
-    interval: Duration,
+#[allow(missing_docs)]
+pub struct AddressActivity {
+    pub start_time: OffsetDateTime,
+    pub interval: Duration,
     addresses: HashSet<Address>,
     // Unfortunately, I don't see another way of implementing it using our current trait design
     flush: Option<usize>,
 }
```

Comment:
It would be nice if the `flush` field comment would also describe what the purpose of this field actually is (I know it's not part of this PR but maybe you can add it).

Reply:
I renamed it to `measurement` in another PR.